react-carousel/src/state/atoms/carouselAtoms.js

```diff
@@ -0,0 +1,39 @@
+import { atom } from 'recoil';
+
+export const slideMovementState = atom({
+  key: '@brainhubeu/react-carousel/slideMovementState',
+  default: {
+    clicked: null, // index of the clicked slide
+    dragStart: 0, // X position of drag event start
+    dragEnd: 0, // X position of drag event end
+    dragOffset: 0, // Distance of the drag
```

Comment:
IMO we should decide to start each of such comments with either lower case or upper case

Reply:
changed to lowercase